x-pack/test/security_solution_endpoint/apps/endpoint/policy_details.ts

```diff
@@ -185,5 +191,41 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
         });
       });
     });
+
+    describe('when on Ingest Configurations Edit Datasource page', async () => {
+      let policyInfo: PolicyTestResourceInfo;
+      beforeEach(async () => {
+        // Create a policy and navigate to Ingest app
+        policyInfo = await policyTestResources.createPolicy();
+        const agentId = policyInfo.agentConfig.id;
+        const datasourceId = policyInfo.datasource.id;
+
+        await pageObjects.common.navigateToApp('ingestManager', {
```

Comment:
I think we spoke about this, and... I might be changing my mind 😃 

I'm thinking that we should create a method in the ingest pageObjects we have (`ingest_manager_create_datasource_page.ts`) named

`navigateToAgentConfigEditDatasource(agentConfigId, datasourceId)`

and call it from here. It will make it easier to keep all ingest related pageObjects in one place, case these move over to Ingest at a later time ++ any future changes to URLs, etc.